### ISSUE TITLE
Make fails on Fedora 24

### ISSUE THREAD

**Issue by limawill** [Ultimaker/cura-build#87]
I am having trouble running the make command.
Here's the terminal output:

```
Checking to see if the QtWebEngineWidgets module should be built...
Checking to see if the dbus support module should be built...
The Python dbus module doesn't seem to be installed.
Qt v5.5.1 (Open Source) is being used.
The qmake executable is
/home/will/Downloads/Cura_Teste/cura-build/build/inst/bin/qmake.
Qt is built as a shared library.
SIP 4.17 is being used.
The sip executable is
/home/will/Downloads/Cura_Teste/cura-build/build/inst/bin/sip.
These PyQt5 modules will be built: QtCore, QtGui, QtNetwork, QtPrintSupport,
QtQml, QtSql, QtSvg, QtTest, QtWidgets, QtXml, QtDBus, QtX11Extras.
The PyQt5 Python package will be installed in
/home/will/Downloads/Cura_Teste/cura-build/build/inst/lib/python3/dist-packages.
PyQt5 is being built with generated docstrings.
PyQt5 is being built with 'protected' redefined as 'public'.
The Designer plugin will be installed in
/home/will/Downloads/Cura_Teste/cura-build/build/inst/lib/plugins/designer.
The qmlscene plugin will be installed in
/home/will/Downloads/Cura_Teste/cura-build/build/inst/lib/plugins/PyQt5.
The PyQt5 .sip files will be installed in
/home/will/Downloads/Cura_Teste/cura-build/build/inst/share/sip.
pyuic5, pyrcc5 and pylupdate5 will be installed in
/home/will/Downloads/Cura_Teste/cura-build/build/inst/bin.
The interpreter used by pyuic5 is /usr/local/bin/python3.4.
Generating the C++ source for the QtCore module...
Embedding sip flags...
Generating the .pro file for the QtCore module...
Generating the C++ source for the QtGui module...
Error: Unable to create the C++ code.
CMakeFiles/PyQt.dir/build.make:110: recipe for target 'PyQt-prefix/src/PyQt-stamp/PyQt-configure' failed
make[2]: *** [PyQt-prefix/src/PyQt-stamp/PyQt-configure] Error 1
CMakeFiles/Makefile2:106: recipe for target 'CMakeFiles/PyQt.dir/all' failed
make[1]: *** [CMakeFiles/PyQt.dir/all] Error 2
Makefile:105: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by Ghostkeeper** [comment]
I've been using Sip v4.18. Maybe that has to do with it?

**Comment by limawill** [comment]
I will try this version!

**Comment by DeuxVis** [comment]
Had the same problem (but on ubuntu Trusty) and fixed it by using PyQT 5.5.1 instead of 5.4.2 :

```
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -292,7 +292,7 @@ if(BUILD_PYQT)
 
     ExternalProject_Add(PyQt
         DEPENDS Python Qt Sip
-        URL http://downloads.sourceforge.net/project/pyqt/PyQt5/PyQt-5.4.2/PyQt-gpl-5.4.2.tar.gz
+        URL http://downloads.sourceforge.net/project/pyqt/PyQt5/PyQt-5.5.1/PyQt-gpl-5.5.1.tar.gz
         PATCH_COMMAND ${pyqt_patch}
         CONFIGURE_COMMAND ${pyqt_command}
         BUILD_IN_SOURCE 1
```

Note that I just can say it fixed the build issue for now, my build is not finished yet so I might have created runtime problems instead...

**Comment by Ghostkeeper** [comment]
We've already switched to 5.6 and we might continue upgrading to 5.7 immediately, so it could then be solved for new people who try to build Cura. Thanks for the report.
